File: IntroPC/Tarea 6/Rutina_primo/rutinas_primo.cpp
#include <cstdio>
int primo (int f)
{
  int i,comp,sw=0;
 
  for(i=2;i<=(f-1);i++)
    {
      
      comp=f%i;
    
      if (comp==0)
	{

	  sw=1;
	  return 0;
	  break;
	  }
    }
  if (sw==0)
    {
      
	return 1;
    }
  
}

File: IntroPC/Tarea 6/Rutina_primo/rutinas_primo.py
#! /usr/bin/env python

def primo(f):
    i=2
    sw=0
    while i<=(f-1):
        comp=f%i
        if comp==0:
            sw=1
            return 0
            break
            
        i+=1
        
    if sw==0:
        return 1
        
    
    
